src/curated-corpus/components/ApprovedItemForm/ApprovedItemForm.tsx

```diff
@@ -103,7 +104,11 @@ export const ApprovedItemForm: React.FC<
       publisher: approvedItem.publisher,
       // A read-only value we may get back from the Pocket Graph
       // for some stories + all collections and syndicated items.
-      datePublished: approvedItem.datePublished ?? null,
+      datePublished: approvedItem.datePublished
+        ? DateTime.fromJSDate(new Date(approvedItem.datePublished)).toFormat(
+            'yyyy-MM-dd',
+          )
+        : null,
```

Comment:
Hm... Does this mean `approvedItem.datePublished` has invalid dates sometimes? The database type is `date`. Or does it perhaps get formatted as `2025-12-11 00:00:00` somewhere by JS?